apps/frontend/src/hooks/useFormsListFiltering.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from 'react';
```

Comment:
Is this file being changed as part of this PR? This feels like it should be for something else and this PR should only focus on the UI.